src/SelectedMovie/SelectedMovie.js

```diff
@@ -1,16 +1,30 @@
-import movieData from '../movie-data'
 import { IoIosArrowDropleft } from "react-icons/io";
 
-const SelectedMovie = ({ id, goBack }) => {
-    const foundMovie = movieData.movies.find(movie => movie.id === id)
+const SelectedMovie = ({ id, goBack, foundMovie }) => {
     return(
         <article>
             <IoIosArrowDropleft onClick={ () => goBack() } role="button"/>
             <img src={foundMovie.backdrop_path} alt={foundMovie.title + 'image'}/>
             <p>{foundMovie.title}</p>
-            <p>{foundMovie.average_rating}</p>
-            <p>{foundMovie.release_date}</p>
+            <p>{'Average Rating: ' + foundMovie.average_rating}</p>
+            <p>{'Released: ' + foundMovie.release_date}</p>
+            <p>{'Budget: $' + foundMovie.budget}</p>
+            <p>{'Genres: ' + foundMovie.genres}</p>
+            <p>{'Overview: ' + foundMovie.overview}</p>
+            <p>{'Revenue: $' + foundMovie.revenue}</p>
+            <p>{foundMovie.tagline}</p>
             <img src={foundMovie.poster_path} alt={foundMovie.title + 'poster'}/>
+            <div className="video-responsive">
```

Comment:
Might be worth checking out https://www.npmjs.com/package/react-youtube for this to play on our site